src/test/java/tests/QualifierTests.java

```diff
@@ -503,5 +530,12 @@ public void statusOrdering() {
         assertEquals(Arrays.asList(4, 5, 6, 7, 0, 1, 2, 3), renderedIssues.stream()
                 .map(TurboIssue::getId)
                 .collect(Collectors.toList()));
+
+        Collections.sort(renderedIssues,
+                Qualifier.getSortComparator(model, "st", false, false));
+        
+        assertEquals(Arrays.asList(0, 1, 2, 3, 4, 5, 6, 7), renderedIssues.stream()
+                .map(TurboIssue::getId)
+                .collect(Collectors.toList()));
```

Comment:
BTW, the three code segments you added seem very similar. Can extract as a method?